src/Illuminate/Database/Query/Grammars/PostgresGrammar.php

```diff
@@ -248,7 +248,7 @@ protected function compileUpdateColumns($query, $values)
         // columns and convert it to a parameter value. Then we will concatenate a
         // list of the columns that can be added into this update query clauses.
         return collect($values)->map(function ($value, $key) {
-            $column = Str::after($key, '.');
+            $column = Arr::last(explode('.', $key));
```

Comment:
There is a global `last()` function that we can use:

```suggestion
            $column = last(explode('.', $key));
```

Reply:
why not just use the PHP `end` function directly? :smile:

Reply:
`end()` only works with variables, not with return values.

Reply:
`last()` is just an alias for `end()` though, or am I missing something?